jx_sqlite/expressions.py

```diff
@@ -258,13 +258,24 @@ def to_sql(self, schema, not_null=False, boolean=False):
 
 @extend(NotOp)
 def to_sql(self, schema, not_null=False, boolean=False):
-    sql = self.term.to_sql(schema)[0].sql
-    return wrap([{"name": ".", "sql": {
-        "0": "1",
-        "b": "NOT (" + sql.b + ")",
-        "n": "(" + sql.n + ") IS NULL",
-        "s": "(" + sql.s + ") IS NULL"
-    }}])
+    if isinstance(self.term, EqOp):
```

Comment:
This can not be allowed. A special case like this indicates something else is wrong with the code. 

This project uses a combination of "symmetry" and test cases to be reasonably sure that the codebase is correct. There are extreme alternatives:

* Proving correctness of the program by analytical means; which humans have proven themselves terrible at
* Test-driven programming - demands an exponential explosion of tests, especially if your code-to-the-test

Between those extremes is a balance between tests and proof - By keeping the code simple and logical (symmetrical), we can reduce the number of tests and still maintain confidence in the code accuracy. The tests do not cover everything, rather they attempt to capturing things that could go wrong.